style.css

```diff
@@ -137,26 +138,35 @@ header {
     border: 1px solid #dfe1e6;
     border-radius: 16px;
     background: #fff;
-    padding: 16px;
+    padding: 10px;
     gap: 12px;
+    margin-left: 9%;
   }
 
   .portfolio-idcard {
     list-style-type: none;
-    margin-left: -3px;
-    margin-top: -3vh;
+    margin-left: 3%;
+    margin-top: -7vh;
   }
 
   .Tonic-1 a {
     text-decoration: none;
     color: #172b4d;
   }
 
+  .tonic {
+    margin-left: 1.5vh;
+  }
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design.
  - you have horizontal overflow so please make sure you remove the horizontal overflow.
  - make sure that you adjust the margin of each project card to look exactly like the Figma design.

| Your Design |Figma Design |
| :---:   | :---: | 
| ![Screenshot from 2023-02-18 16-18-50](https://user-images.githubusercontent.com/92303891/219877964-76529fd5-ee7b-46da-bba8-d7d2542e7e25.png)| ![image](https://user-images.githubusercontent.com/92303891/219878165-8fc7796f-40cd-4e70-9921-29b1919d9dc2.png) |